lib/models/builder.js

```diff
@@ -12,15 +12,26 @@ var chalk       = require('chalk');
 var attemptNeverIndex = require('../utilities/attempt-never-index');
 var findBuildFile = require('../utilities/find-build-file');
 
-var FSMonitor = require('heimdalljs-fs-monitor');
-var monitor = new FSMonitor();
-monitor.start();
-
 var Sync = require('tree-sync');
 
 var signalsTrapped = false;
 var buildCount = 0;
 
+function _enableFSMonitorIfVizEnabled() {
+  var FSMonitor = require('heimdalljs-fs-monitor');
```

Comment:
Hmm. OK.  Seems fine either way, just didn't seem like we needed to even require it if it wasn't needed.